cmd/wio/commands/run/run.go

```diff
@@ -109,6 +116,32 @@ func (info *runInfo) clean(targets []types.Target) error {
 }
 
 func (info *runInfo) build(targets []types.Target) error {
+    // checks for source files based on header only and package
+    if info.projectType == constants.PKG {
+        var path string
```

Comment:
is this really necessary? Cmake will give the same errors anyway

Reply:
yeah but that looks kinda ugly we should handle these cases ourselves. This way user gets a nice message

Reply:
could you add the logic for this in a separate function call then, it's kinda cluttering up the build function